src/graph.ts

```diff
@@ -15,7 +15,7 @@ export class ContainerNode {
      * this container, and a set of consumers is drawing from this container.
      */
     readonly producers = new Set<IndustryNode>()
-    readonly consumers = new Set<IndustryNode>()
+    readonly consumers = new Set<ConsumerNode | OutputNode | IndustryNode>()
```

Comment:
Since both `OutputNode` and `IndustryNode` extend `ConsumerNode`, this is sufficient
```suggestion
    readonly consumers = new Set<ConsumerNode>()
```

Reply:
Fixed in https://github.com/lgfrbcsgo/du-factory-generator/commit/2f4a32d097a7f59ef06a5d257c5e7cee14db819e